src/Sparrow/Json/JsonOperationContext.cs

```diff
@@ -106,7 +107,7 @@ public unsafe LazyStringValue AllocateStringValue(string str, byte* ptr, int siz
             }
 
             var allocateStringValue = new LazyStringValue(str, ptr, size, this);
-            if (_numberOfAllocatedStringsValues < 32 * 1_024)
+            if (_numberOfAllocatedStringsValues < _maxNumberOfAllocatedStringValues)
```

Comment:
Each context in a pool can keep `_maxNumberOfAllocatedStringValues * sizeof(LazyStringValue)` where `sizeof(LazyStringValue)` is 80 bytes => 2.6MB